docker-compose.yml

```diff
@@ -44,6 +44,21 @@ services:
       timeout: 5s
       retries: 5
 
+
+
+  gotenberg:
```

Comment:
meh.. couple of extra lines here.. if you're doing a follow up could tidy this up?